### DISCUSSION TITLE
Interactive Inputs

### DISCUSSION THREAD

**Discussion by neilharris1** [argoproj/argo-workflows#6789] [General]
I've been reading through the documentation about inputs in argo workflows. So far I've only seen information about how to provide input arguments to workflow through the command line. 

What I would like to know is, if there is any way you can create a workflow that providess an interactive input. For example, a text box where you can type the input parameter before executing the workflow (similar to what is available in jenkins pipelines)?

**Comment by tczhao** [comment]
Have a look at workflow template
Is this what you were asking?
![image](https://user-images.githubusercontent.com/20961507/135547925-390a46ed-c59f-4c51-88e8-64502e2aa73a.png)
